src/dag-node.js

```diff
@@ -82,6 +82,9 @@ module.exports = class DAGNode {
 
   // addNodeLink - adds a DAGLink to this node that points to node by a name
   addNodeLink (name, node) {
+    if (!name) {
```

Comment:
What is? go-ipfs doesn't accept empty arrays or `undefined` (doesn't even exist in Go), so JS certainly shouldn't.